python/cudf/cudf/utils/utils.py

```diff
@@ -396,10 +396,10 @@ def pa_mask_buffer_to_mask(mask_buf, size):
     return Buffer(mask_buf)
 
 
-def search_range(n, start, step, length, side="left"):
+def search_range(start, stop, x, step=1, side="left"):
     """Search the position to insert a value in a monotonically increasing
-    integral arithmatic series so that it will not change the sortedness of the
-    series.
+    integral arithmatic series (represented by range), so that it will not
+    change the sortedness of the series.
```

Comment:
Let's keep this simple:

```suggestion
    """Find the position to insert a value in a range, so that the resulting sequence
       remains sorted.
```